waiter/src/waiter/core.clj

```diff
@@ -88,14 +88,15 @@
                      "service-id" :service-id-handler-fn
                      "settings" :display-settings-handler-fn
                      "sim" :sim-request-handler
-                     "state" {"" :state-all-handler-fn
-                              "/kv-store" :state-kv-store-handler-fn
-                              "/leader" :state-leader-handler-fn
-                              "/maintainer" :state-maintainer-handler-fn
-                              "/router-metrics" :state-router-metrics-handler-fn
-                              "/scheduler" :state-scheduler-handler-fn
-                              "/statsd" :state-statsd-handler-fn
-                              ["/" :service-id] :state-service-handler-fn}
+                     "state" [["" :state-all-handler-fn]
+                              ["/kv-store" :state-kv-store-handler-fn]
+                              ["/last-request-times" :state-last-request-times-handler-fn]
+                              ["/leader" :state-leader-handler-fn]
+                              ["/maintainer" :state-maintainer-handler-fn]
+                              ["/router-metrics" :state-router-metrics-handler-fn]
+                              ["/scheduler" :state-scheduler-handler-fn]
+                              ["/statsd" :state-statsd-handler-fn]
+                              [["/" :service-id] :state-service-handler-fn]]
                      "status" :status-handler-fn
                      "token" :token-handler-fn
                      "tokens" { "" :token-list-handler-fn
```

Comment:
We should change **all** of these map literals to vectors-of-vectors so that we can ensure the order. That way when the `"apps"` entry above crosses the threshold from 8 to 9 somewhere down the road, we don't get the same weird behavior again. Since that threshold number (8 or 9 or whatever) is an implementation detail, we really shouldn't be relying on that anyway.

Reply:
That can be part of another PR, I don't want to extend the scope of this PR